layouts/joomla/content/intro_image.php

```diff
@@ -22,27 +22,31 @@
 	return;
 }
 
-$imgfloat = empty($images->float_intro) ? $params->get('float_intro') : $images->float_intro;
+$imgfloat  = empty($images->float_intro) ? $params->get('float_intro') : $images->float_intro;
+$extraAttr = '';
+$img       = HTMLHelper::cleanImageURL($images->image_intro);
 
-$img = HTMLHelper::cleanImageURL($images->image_intro);
-
-// Set or not the lazyloading
-$img->attributes['loading'] = 'lazy';
+// Set lazyloading only for images that there's a record of width and height
+if ((isset($img->attributes['width']) && (int) $img->attributes['width'] > 0)
+		&& (isset($img->attributes['width']) && (int) $img->attributes['width'] > 0))
```

Comment:
```suggestion
&& (isset($img->attributes['height']) && (int) $img->attributes['height'] > 0))
```
Width has already been checked in the previous line, remove indent